app/services/groups/transfer_service.rb

```diff
@@ -19,8 +19,13 @@ def execute(new_namespace)
         raise TransferError, I18n.t('services.groups.transfer.namespace_group_exists')
       end
 
+      group_ancestor_members_user_ids = Member.for_namespace_and_ancestors(@group).map(&:user_id)
```

Comment:
We should use `.select(:user_id)` over `map(&:user_id)` here. `select` will get just the `user_id` column for the table, whereas `map` gets all the columns and then gets the `user_id`s

You can try this in rails console to see the difference in the SQL statement

`group = Group.first`
`Member.for_namespace_and_ancestors(group).select(:user_id)`
`Member.for_namespace_and_ancestors(group).map(&:user_id)`

Reply:
Ok. 

![Screenshot from 2023-10-26 14-18-39](https://github.com/phac-nml/irida-next/assets/325703/546f1271-8420-4a8d-99ab-dbff5f3d3fe4)

Reply:
You can just use the result of `.select(:user_id)`. Should not have to use pluck

Reply:
Please see d72ff36bb8eaca2c0a460fb9ace62ec6f1447beb.